sdk/sdk.yaml

```diff
@@ -2943,14 +2943,14 @@ paths:
           content:
             application/json:
               schema:
-                $ref: ../evm_responses.yaml#/debug_traceCall
+                $ref: ./evm_responses.yaml#/debug_traceCall
       operationId: debug-tracecall
```

Comment:
the operation id should be sdk-debug-tracecall 

otherwise it will be the same as the non-sdk debug-tracecall and this will break links in docs. This applies to all sdk debug methods